spec/tictactoe/game_spec.rb

```diff
@@ -1,190 +1,303 @@
+      validate = FakeValidation.new
+      new_game = Tictactoe::Game.new(validate)
+      moves = [
+        ["X", 3], #
+        ["O", 4],
+        ["X", 6], #
+        ["O", 7],
+        ["X", 9], #
+      ]
+
+      moves.each do |symbol, position|
+        new_game.make_move(position, symbol)
+      end
+      expect(new_game.winning_combinations("X")).to eq (true)
+    end
+    it "should check for seventh winning combinations" do
+      validate = FakeValidation.new
+      new_game = Tictactoe::Game.new(validate)
+      moves = [
+        ["X", 1], #
+        ["O", 2],
+        ["X", 5], #
+        ["O", 7],
+        ["X", 9], #
+      ]
+
+      moves.each do |symbol, position|
+        new_game.make_move(position, symbol)
       end
-      it "should check for fifth winning combinations" do
-        new_game = Tictactoe::Game.new
-        moves = [
-          ["X", 2], #
-          ["O", 3],
-          ["X", 5], #
-          ["O", 7],
-          ["X", 8], #
-        ]
-
-        moves.each do |symbol, position|
-          new_game.play(position, symbol)
-        end
-        expect(new_game.winning_combinations("X")).to eq (true)
+      expect(new_game.winning_combinations("X")).to eq (true)
+    end
+    it "should check for eigth winning combinations" do
+      validate = FakeValidation.new
+      new_game = Tictactoe::Game.new(validate)
+      moves = [
+        ["X", 3], #
+        ["O", 2],
+        ["X", 5], #
+        ["O", 1],
+        ["X", 7], #
+      ]
+
+      moves.each do |symbol, position|
+        new_game.make_move(position, symbol)
       end
-      it "should check for sixth winning combinations" do
-        new_game = Tictactoe::Game.new
-        moves = [
-          ["X", 3], #
-          ["O", 4],
-          ["X", 6], #
-          ["O", 7],
-          ["X", 9], #
-        ]
-
-        moves.each do |symbol, position|
-          new_game.play(position, symbol)
-        end
-        expect(new_game.winning_combinations("X")).to eq (true)
+      expect(new_game.winning_combinations("X")).to eq (true)
+    end
+  end
+
+  context "get_position" do
+    it "should output 'Enter position' the console" do
+      input = FakeStringInput.new(["1"])
+      validate = Tictactoe::Validation.new
+      new_game = Tictactoe::Game.new(validate, stdin: input)
+      expect { new_game.get_position }.to output("Enter position\n").to_stdout
+    end
+    it "should accept input from the console" do
+      input = FakeStringInput.new(["4"])
+      validate = Tictactoe::Validation.new
+      new_game = Tictactoe::Game.new(validate, stdin: input)
+      expect(new_game.get_position).to eq(4)
+    end
+    it "should print error message if position input is invalid" do
+      validate = Tictactoe::Validation.new
+      input = FakeStringInput.new(["20", "3"])
+      new_game = Tictactoe::Game.new(validate, stdin: input)
+      expect {
+        symbol = new_game.get_position
+        expect(symbol).to eq(3)
```

Comment:
Since accepting input is tested on line 240, I think we only need to assert that we show the out of range error.